src/server/app/admin/services/users.profile.server.service.js

```diff
@@ -39,3 +39,11 @@ module.exports.userById = (id) => {
 
 	return defer.promise;
 };
+
+module.exports.updateUser = function(query) {
+	return User.update(query).exec();
+};
+
+module.exports.searchAll = function(query) {
```

Comment:
This may already be provided by `User.search` with the variety of other options.